ccd-config-generator/src/main/java/uk/gov/hmcts/ccd/sdk/type/Document.java

```diff
@@ -24,14 +24,19 @@ public class Document {
   @JsonProperty("document_binary_url")
   private String binaryUrl;
 
+  @JsonProperty("category_id")
+  private String categoryId;
+
   @JsonCreator
```

Comment:
@sjamihmcts Changing this constructor signature is a breaking change, this should have been a constructor overload that takes the new param.